crates/bevy_gltf/src/lib.rs

```diff
@@ -267,11 +270,14 @@ impl Plugin for GltfPlugin {
         let default_sampler = default_sampler_resource.get_internal();
         app.insert_resource(default_sampler_resource);
 
+        let extensions = app.world().resource::<GltfExtensionHandlers>();
+
         app.register_asset_loader(GltfLoader {
             supported_compressed_formats,
             custom_vertex_attributes: self.custom_vertex_attributes.clone(),
             default_sampler,
             default_use_model_forward_direction: self.use_model_forward_direction,
+            extensions: extensions.0.clone(),
```

Comment:
I don't like this. The way it's written right now, modifying `GltfExtensionHandlers` after the app starts is a no-op, but there's no indication of that. How about storing an `Arc<async_lock::RwLock<Vec<...>>>` for the handlers instead? That way, the gltf loader can just lock the RwLock and then clone the inner handlers. Then users can in theory modify the handlers at runtime if they want!